pyomo/contrib/fbbt/interval.py

```diff
@@ -471,7 +471,7 @@ def asin(xl, xu, yl, yu):
         # satisfies xl = sin(y)
         lb1 = i1 + dist
         lb2 = i2 + dist
-        if lb1 >= yl:
+        if lb1 >= yl - feasibility_tol:
```

Comment:
I feel like the sign on these feasibility tolerance adjustments is reversed.  consider:
```python
feasibility_tol = 0.01
yl = 1.0
lb1 = 1.00001
```
then
```python
if 1.00001 > 1 - 0.01:
    lb = 1.00001 # ?!?
```